src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import "./App.css";
 import { Outlet } from "react-router-dom";
-import Header from "./components/header";
+import Header from "./components/Header";
```

Comment:
You changed this import to capital H Header but the file is still lowercase h header so the app failed to start. Can you just make the change and then update this commit?

TODO for ryan: I'll make a video example of how to update this commit

Reply:
So you can fix this issue and update this most recent commit so that you dont need to make a new commit to fix the branch with the command `git commit --amend` which updates the latest commit with a change rather than creating a new commit.

I made a little video to show you how you would do this and Ill make some git notes at some point and include git commit --amend

Heres a link to the example video:
https://drive.google.com/file/d/1oKS1D9l5ISVsAeTnFsY3MDmGLFbXZI-b/view?usp=drive_link

Reply:
If you've already made more commits on your machine that arent in the PR yet they don't worry about this. That makes it more complicated